### ISSUE TITLE
MPD: Login attempt or request with invalid authentication 2022.4.1

### ISSUE THREAD

**Issue by Chuppa** [Poeschl/Hassio-Addons#310]
**Which addon?**
MPD

**Describe the bug**
Notification in HA each time a media file is played.  Did not occur prior to 2022.4

```
ERROR (MainThread) [homeassistant.components.automation.house_announce_time] House: Announce Time: Error executing script. Error for call_service at pos 2: Entity media_player.mpd does not support this service.
ERROR (MainThread) [homeassistant.components.automation.house_announce_time] Error while executing automation automation.house_announce_time: Entity media_player.mpd does not support this service.
ERROR (MainThread) [frontend.js.latest.202204050] :0:0 Script error.
WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 243ffc37-mpd.local.hass.io (172.30.33.2). (Music Player Daemon 0.22.3)
```
**To Reproduce**
Play media file, notification appears in HA.

**System**

* Supervisor version: supervisor-2022.03.5
* Host system version: Home Assistant OS 7.6
<img width="485" alt="Screen Shot 2022-04-11 at 01 47 57" src="https://user-images.githubusercontent.com/1456333/162627912-e5bf7143-bac4-4a37-b3fe-7633a18dfe7a.png">

**Comment by Poeschl** [comment]
This might have something to do with the Mpd integration and not the addon. Please look there for help.

**Comment by Chuppa** [comment]
Issue is resolved in 2022.4.3
